sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson

```diff
@@ -906,7 +906,7 @@
             '''
       tags: ["rom", "no_dv", "fpga", "silicon"]
       stage: V2
-      tests: []
+      tests: ["//sw/device/silicon_creator/rom/e2e:rom_e2e_asm_interrupt_handler"]
```

Comment:
This should just be the name of the test gets added to the dvsim config file [`chip_rom_tests.hjson`](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/dv/chip_rom_tests.hjson;drc=0ccf5d9703064f2eaa19ece04ea63021783f4508;l=12). Updating this field populates the nightly [dashboard](https://reports.opentitan.org/hw/top_earlgrey/dv/latest/report.html). Since these tests have not been added to DV yet, these fields should be empty for now. #14634 is tracking adding these to DV.

#15812 should fix. Lmk if you have any questions.